tests/fixtures/preprocessor-tests/app-with-addon-without-preprocessors/node_modules/broccoli-sass/index.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+var Writer = require('broccoli-writer');
+
+function copyPreserveSync (src, dest) {
```

Comment:
@stefanpenner This is a dummy SASS preprocessor that's used by the tests, each test included it  in `node_modules/`. I was thinking it would be best to abstract it out somehow, since the new acceptance helper links against modules that already exist, but I wasn't sure what the best way to do it would be. For now I removed the second copy since it'll already be in `node_modules-tmp/`, will try to think of a better way to abstract out shared modules for tests in the future.